src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -253,7 +253,7 @@ private function addFormSection(ArrayNodeDefinition $rootNode, callable $enableI
                             ->children()
                                 ->scalarNode('enabled')->defaultNull()->end() // defaults to framework.csrf_protection.enabled
                                 ->scalarNode('token_id')->defaultNull()->end()
-                                ->scalarNode('field_name')->defaultValue('_token')->end()
+                                ->scalarNode('field_name')->defaultValue('_csrf_token')->end()
```

Comment:
I am still 👎 on this change as I don't think it's worth it potentially breaking applications just to change the name of this field.

If we really want to change the default name we should IMO follow the usual deprecation-first approach: Meaning here that not setting the value explicitly would trigger a deprecation which would then allow us to change the default in 8.0.